### DISCUSSION TITLE
Discovery Page is broken after using filters

### DISCUSSION THREAD

**Discussion by Pr0mises** [community/community#136062] [Mobile]
I use the mobile Github app for quite a while now. First of all good job, the app is awesome (saving the written text would be great tho).

Android: 14 (S22+)
Github App version: 1.173.0

Now to my issue.

Few days ago I went to the discovery tab as usual to get the latest updates on releases, new repos or discussions etc. and noticed a filter button.
I unchecked a few optioms and was left with an empty page.
I checked all options again and only receive the following feed since then:
- Repos created, starred or forked by people I follow
- recommendation
which results in an pretty empty discovery tab.

I tried deleting the cache and data of the app, relogged few times, deleted the app, restarted my phone but the issue persists.

Does anyone know how I can fix that?

Thanks in advance

**Comment by ItsXcr9** [comment]
can you attach the screen shots?

**Reply by Pr0mises** [reply]
Yes, sry about that. I edited my post

**Comment by Pr0mises** [comment]
Alright so I deselected all filters again which results in this:
![Screenshot_20240819_193547_GitHub.jpg](https://github.com/user-attachments/assets/0cb5c647-6d81-4002-be64-359f33975553)

After selecting all filters again, the last 2 entries in last screenshot posted above disappeared. Somehow my feed is getting smaller and nothing new (besides of recommendation) appears...

**Comment by Pr0mises** [comment]
I was able to fix the issue.
I'm not sure what happened in the first place but as I wasn't able to find the filter option in the desktop mode of github via browser as well.

I just opened the mobile version in the browser -> github -> filter -> changing filters didn't work - There are also more filters via the browser than via the mobile app

But this worked:
Browser -> github.com -> Filter -> Reset to default


Everything is working again and I'm able to change the filters in the mobile app again


Edit: nvm changing the filters via the mobile app works for a few minutes, restarting the app breaks the discovery again. Using "Reset to defaults" via the mobile version in my browser fixes it tho

**Comment by Pr0mises** [comment]
It looks like the bug still persist even without changing the filters. Therefore I unmarked my solution, it's a fix but the problem still persists

**Reply by talaqalotaibi** [reply]
- [ ] #![image](https://github.com/user-attachments/assets/f0bdeefd-a8b8-4d40-b1e2-b7d83dc8331c)

**Comment by Bloopps** [comment]
Same issue here discovery page dont work !
![IMG_20241107_182449.jpg](https://github.com/user-attachments/assets/13fb80d6-c4c5-4964-ab60-3555ad057334)

**Comment by xximj** [comment]
Same here. +1

**Reply by talaqalotaibi** [reply]
ATS

**Comment by Bloopps** [comment]
Please do something, I can't see any new releases on the repositories I follow. I can no longer follow the news![Screenshot_2024-12-06-13-23-48-92_320a9a695de7cdce83ed5281148d6f19.jpg](https://github.com/user-attachments/assets/f4821d68-6784-4750-8b1f-85258c5dbfc5)

**Reply by isi07** [reply]
+1 same issue here

**Reply by N3rdix** [reply]
Same here... very annoying
